uint8 rightMotorSpeed
bool rightMotorSpeedCommand
uint8 leftMotorSpeed
bool leftMotorSpeedCommand
int16 cameraTiltShift
bool cameraTiltShiftCommand
int16 cameraPanShift
bool cameraPanShiftCommand
uint8 digital1
bool digital1Command
uint8 digital2
bool digital2Command
---
bool commandSuccessful
